test/LibraryManager.IntegrationTest/Services/InstallDialogTestService.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using Microsoft.Test.Apex.Services;
+using Microsoft.Test.Apex.VisualStudio;
+using Microsoft.Test.Apex.VisualStudio.Shell;
+using Microsoft.Web.LibraryManager.Vsix.UI;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest.Services
+{
+    /// <summary>
+    /// Test service for add client side libraries dialog.
+    /// </summary>
+    [Export(typeof(InstallDialogTestService))]
+    public class InstallDialogTestService : VisualStudioTestService
```

Comment:
>InstallDialogTestService [](start = 17, length = 24)

Normally we put our TestService and TestExtension classes into a separate DLL since the assembly containing them gets loaded in-proc in VS.  I know we had some issues long ago where putting them in the test case assembly caused weird MEF issues with Apex, I'm not sure if that design problem was ever resolved.

Reply:
We can tackle this one later, if it actually produces issues.

---
In reply to: [206626060](https://github.com/aspnet/LibraryManager/pull/332#discussion_r206626060) [](ancestors = 206626060)